### ISSUE TITLE
Nullpointrer exception on reconnection

### ISSUE THREAD

**Issue by andreafretti** [LennysLounge/ACC-Race-Control#19]
If I disconnect from a server (using the disconnection button) and reconnect I get tge following NullPointer exception:

```
java.lang.NullPointerException
	at racecontrol.client.extension.autobroadcast.RatingProcessorImpl.calculateRating(RatingProcessorImpl.java:88)
	at racecontrol.client.extension.autobroadcast.AutobroadcastExtension.updateCarRatings(AutobroadcastExtension.java:173)
	at racecontrol.client.extension.autobroadcast.AutobroadcastExtension.onSessionUpdate(AutobroadcastExtension.java:117)
	at racecontrol.client.extension.autobroadcast.AutobroadcastExtension.onEvent(AutobroadcastExtension.java:91)
	at racecontrol.eventbus.EventBus.lambda$publish$0(EventBus.java:36)
	at java.base/java.lang.Iterable.forEach(Iterable.java:75)
	at racecontrol.eventbus.EventBus.publish(EventBus.java:35)
	at racecontrol.client.AccConnection.onRealtimeUpdate(AccConnection.java:372)
	at racecontrol.client.protocol.AccBroadcastingProtocol.readRealtimeUpdate(AccBroadcastingProtocol.java:147)
	at racecontrol.client.protocol.AccBroadcastingProtocol.processMessage(AccBroadcastingProtocol.java:79)
	at racecontrol.client.AccConnection.run(AccConnection.java:150)
```

The app do not crashed but it is not working anymore

**Comment by LennysLounge** [comment]
Thank you very much for reporting this issue.

Ill look into it and try to fix it as fast as i can

Am Fr., 21. Apr. 2023 um 17:47 Uhr schrieb andreafretti <
***@***.***>:

**Comment by LennysLounge** [comment]
Closed with the 1.5.3 release
